activebridge/CabLook-backend
# Add email verification

Add `email_verification_token` to users
- controller: Verifications::EmailsController
- actions: 
  - GET `/verifications/send_verification`
  - POST `/verifications/verify`
- event classes: 
  - Events::Verifications::SendEmailVerification 
    - data: { email: required };
    - generate email_verification_token;
    -  send verification mail to user email.
  - Events::Verifications::VerifyEmail
    -  data: { email: required, verification_code: required };
    - find user by email;
    - check to equals email_verification_token.
